src/EA.Weee.RequestHandlers/Organisations/DirectRegistrants/CompleteOrganisationTransactionHandler.cs

```diff
@@ -64,7 +64,19 @@ public async Task<Guid> HandleAsync(CompleteOrganisationTransaction request)
 
                     var brandName = await CreateAndAddBrandName(organisationTransactionData);
 
-                    var directRegistrant = DirectRegistrant.CreateDirectRegistrant(organisation, brandName);
+                    var contactDetails = new Contact(organisationTransactionData.ContactDetailsViewModel.FirstName, organisationTransactionData.ContactDetailsViewModel.LastName,
```

Comment:
@will-hope98 Just to tidy the code can we move the creation of the objects into local private methods similar to how the others are created

Reply:
Done